android/src/androidTest/java/com/reactnativestripesdk/addresssheet/AddressSheetViewTest.kt

```diff
@@ -178,7 +178,7 @@ class AddressSheetViewTest {
         ),
       )
 
-    val expected =
+    val result =
```

Comment:
I had to change the shape of a few events so the payload is an object to be able to use UnsafeMixed codegen. This doesn't change the public api.